providers/plunk/src/lib/plunk.provider.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  IEmailOptions,
+  IEmailProvider,
+  ICheckIntegrationResponse,
+  IEmailEventBody,
+  CheckIntegrationResponseEnum,
+} from '@novu/stateless';
+
+import Plunk from '@plunk/node';
+import { IPlunkResponse } from './plunk.interface';
+
+export class PlunkEmailProvider implements IEmailProvider {
+  id = 'plunk';
+  channelType = ChannelTypeEnum.EMAIL as ChannelTypeEnum.EMAIL;
+
+  private plunk: Plunk;
+
+  constructor(
+    private config: {
+      apiKey: string;
+    }
+  ) {
+    this.plunk = new Plunk(this.config.apiKey);
+  }
+  getMessageId?: (body: any) => string[];
+  parseEventBody?: (body: any, identifier: string) => IEmailEventBody;
```

Comment:
Seem unnecessary. 🤔

Reply:
I think this was generated in the boilerplate. I've updated it any ways.